js/src/product-feed/index.js

```diff
@@ -10,18 +11,45 @@ import NavigationClassic from '.~/components/navigation-classic';
 import IssuesTableCard from './issues-table-card';
 import ProductFeedTableCard from './product-feed-table-card';
 import SubmissionSuccessGuide from './submission-success-guide';
+import CustomerEffortScorePrompt from '.~/components/customer-effort-score-prompt';
 import ProductStatistics from './product-statistics';
 import './index.scss';
 import useProductFeedPrefetch from './useProductFeedPrefetch';
 import AppSpinner from '.~/components/app-spinner';
+import { GUIDE_NAMES, LOCAL_STORAGE_KEYS } from '.~/constants';
+import localStorage from '.~/utils/localStorage';
+import isWCTracksEnabled from '.~/utils/isWCTracksEnabled';
 
 const ProductFeed = () => {
 	const { hasFinishedResolution, data } = useProductFeedPrefetch();
 
+	// Show submission success guide modal by visiting the path with a specific query `guide=submission-success`.
+	// For example: `/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fproduct-feed&guide=submission-success`.
+	const isSubmissionSuccessOpen =
+		getQuery().guide === GUIDE_NAMES.SUBMISSION_SUCCESS;
```

Comment:
💅  Again, Personal opinion 100% not blocking

I saw that `getQuery()` return always Object somehow.  But maybe `getQuery()?.guide` would be safer, since it's an external dependency.

Reply:
Great suggestion! Added in 19ff496.